app/site/controllers.py

```diff
@@ -109,7 +106,8 @@ def datapackage_show(publisher, package):
         .split('\n\n')[0].replace(' \n', '') \
         .replace('\n', ' ').replace('/^ /', '')
 
-    return render_template("dataset.html", dataset=dataset,
+    return render_template("dataset.html", user=user,
```

Comment:
User (or better something a bit more descriptive like `current_user`) should not be set in every method but injected globally into the tmeplates see e.g. http://stackoverflow.com/questions/26498689/flask-jinja-pass-data-to-a-base-template-all-templates